src/did_resolve.rs

```diff
@@ -676,9 +675,13 @@ impl DIDResolver for HTTPDIDResolver {
             }
         };
         let https = HttpsConnector::new();
+
+        static USER_AGENT: &str = concat!(env!("CARGO_PKG_NAME"), "/", env!("CARGO_PKG_VERSION"));
+
         let client = Client::builder().build::<_, hyper::Body>(https);
         let request = match Request::get(uri)
             .header("Accept", TYPE_DID_RESOLUTION)
+            .header("User-Agent", USER_AGENT)
```

Comment:
This could use the crate `USER_AGENT` constant now, so the local `USER_AGENT` constant is no longer needed.
```suggestion
        let client = Client::builder().build::<_, hyper::Body>(https);
        let request = match Request::get(uri)
            .header("Accept", TYPE_DID_RESOLUTION)
            .header("User-Agent", crate::USER_AGENT)
```